src/quick-tour/catalog-page.md

```diff
@@ -2,7 +2,7 @@
 title: Catalog Page
 ---
 
-Catalog page listings typically have small product images and brief descriptions, and can be formatted as a list or as a grid. You can add banners, videos, and keyword-rich descriptions, and also create special designs for a promotion or season. You might create a special category to feature a lifestyle or brand that is a curated collection of products from different categories.
+Catalog page listings typically have small product images and brief descriptions, and can be formatted as a list or as a grid. You can add blocks, videos, and keyword-rich descriptions, and also create special designs for a promotion or season. You might create a special category to feature a lifestyle or brand that is a curated collection of products from different categories.
```

Comment:
should this be "dynamic blocks"?
maybe with a topic link?

Reply:
I'm going to leave it simple for now. It might be just the regular PB blocks, or both.